src/azure-cli/HISTORY.rst

```diff
@@ -16,10 +16,10 @@ Release History
 **ACR**
 
 * Support zone redundancy (#15975)
-* `az acr connected-registry`: New feature for on-prem Azure Container Registry (#16238)
-* `az acr scope-map update`: --add and --remove are deprecated, they are renamed to --add-repo --remove-repo (#16238)
-* `az acr scope-map create/update`: Add support to handle Gateway actions. (#16238)
-* `az acr token create`: support added for gateway actions (#16238)
+* `az acr connected-registry`: Added support for private preview of connected registry feature. (#16238)
```

Comment:
hi @fengzhou-msft will this manual updating  be overwrite by next release?

Reply:
@fengzhou-msft  @yungezz I'm not really sure what is the right procedure here. Is this PR ok?

Reply:
Release notes of previous versions will not be overriden. @rosanch you may also need to submit a PR in [docs repo](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/docs-ref-conceptual/release-notes-azure-cli.md).

Please use **present-tense** verbs to keep consistent, e.g. use `add` instead of `added`.